pkg/analyzer/analyzers/elevenlabs/elevenlabs.go

```diff
@@ -0,0 +1,483 @@
+package elevenlabs
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"slices"
+	"strings"
+	"sync"
+
+	"github.com/fatih/color"
+	"github.com/google/uuid"
+	"github.com/jedib0t/go-pretty/table"
```

Comment:
```suggestion
	"github.com/jedib0t/go-pretty/v6/table"
```

`github.com/jedib0t/go-pretty/v6` is the latest version with bug fixes and enhancements. See also https://github.com/trufflesecurity/trufflehog/pull/3924